documentation/book/assembly-distributed-tracing.adoc

```diff
@@ -0,0 +1,40 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-distributed-tracing-{context}']
+= Distributed tracing
+
+This chapter provides an overview of distributed tracing in {ProductName}. It includes procedures for configuring distributed tracing in Kafka Connect and Mirror Maker, and for instrumenting Kafka Producer, Consumer, and Streams API applications for tracing.
```

Comment:
Is 'instrumenting' the same as configuring?

Reply:
Good point. It was difficult to decide whether to start with general info/definitions for tracing, or with the product-specific details.

Whilst I expect most developers will know a little about instrumentation, I've added a brief explanation.